plugins/inputs/opentelemetry/opentelemetry_test.go

```diff
@@ -86,7 +86,7 @@ func TestOpenTelemetry(t *testing.T) {
 		exesuffix = ".exe"
 	}
 	expected := []telegraf.Metric{
-		tmetric.New(
+		telegraf_metric.New(
```

Comment:
Another way could also be this:
```suggestion
		testutil.MustMetric(
```

Reply:
Good point, changed.